resources/lib/utils/logger.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is unused so can be removed.

Personally I debug using the method described here: https://kodi.wiki/view/HOW-TO:Debug_Python_Scripts_with_Web-PDB
So temporarily adding web-pdb as dependency and triggering that from the code.

But adding some form of actual logging would be fine too.

Reply:
If you're running Kodi solely on Android then using web-pdb might be a bit harder indeed.